content/en/news/release3.5.md

```
@@ -1,163 +1,186 @@
-title: OpenMS 3.5 released
+title: "OpenMS 3.5 Released"
 authors: ["Tjeerd Dijkstra"]
 date: 2025-12-11
-summary: We have just released OpenMS 3.5.0
+summary: "We have just released OpenMS 3.5.0"
 ---
+
 Dear OpenMS-Users,
 
-we are proud to announce the release of OpenMS 3.5. Grab it <a href="https://github.com/OpenMS/OpenMS/releases/tag/release%2F3.5.0">here</a>
+We are proud to announce the release of OpenMS 3.5. Grab it [here](https://github.com/OpenMS/OpenMS/releases/tag/release%2F3.5.0).
```

markdownlint-cli2 (0.20.0)

Link text should be descriptive

(MD059, descriptive-link-text)